gemrb/core/GUI/GameControl.cpp

```diff
@@ -2038,6 +2028,33 @@ bool GameControl::HandleActiveRegion(InfoPoint *trap, Actor * actor, const Point
 	return false;
 }
 
+// Calculate the angle between a clicked point and the first selected character,
+// so that we can set a sensible orientation for the formation
+void GameControl::InitFormation (const Point &clickPoint)
+{
+	/* We get called on initial click, but also on drag.  Perform the setup only once.  */
```

Comment:
this comment appears to be incorrect

Reply:
Ah, that was from before I moved the Alt-modifier stuff to also happen on mouse down.